jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -417,6 +458,37 @@ String getFormat() {
     return Preconditions.checkNotNull(container.format);
   }
 
+  /**
+   * Gets the configured extra directory path.
+   *
+   * @return the configured extra directory path
+   */
+  @Nullable
+  Path getExtraDirectoryPath() {
+    // TODO: Should inform user about nonexistent directory if using custom directory.
+    if (System.getProperty(PropertyNames.EXTRA_DIRECTORY_PATH) != null) {
+      return Paths.get(System.getProperty(PropertyNames.EXTRA_DIRECTORY_PATH));
```

Comment:
Should check the Maven settings and pom properties too to keep with the semantics of Maven's `@Parameter(property)` values?

Reply:
Is this a trivial change? It sounds like something we may want to do for all of the parameters, so maybe it should wait until another PR.

Reply:
Speaking of which, I think supporting globs would be a good idea, but I think that should wait until another PR as well.

Reply:
Filed #1200 and #1201 for these 2 comments.